docker/Dockerfile.dist.ubuntu14.04

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get -y install \
+      curl \
+      libssl-dev \
+      git \
```

Comment:
Do you mind sorting these alphabetically, it helps avoid accidentally adding a dependency twice. 

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/